spec/requests/case_workers/claims/court_data_spec.rb

```diff
@@ -6,6 +6,8 @@
     allow(LAA::Cda::ProsecutionCase).to receive(:search).and_return([])
   end
 
+  # after { sign_out user.user }
```

Comment:
Can this be removed now? It's the only change in [this commit](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/8202/commits/617e8f5a9f3c4af07b3e38286a2403b76aa9d39f), so possibly the whole commit could be deleted?